__tests__/unit/core-transaction-pool/connection.test.ts

```diff
@@ -241,16 +239,19 @@ describe("Connection", () => {
             expect(notAdded).toBeEmpty();
 
             expect(connection.getPoolSize()).toBe(4);
-            await delay((expireAfterSeconds + 1) * 1000);
+
+            /* XXX how to wait until the current height becomes `expiration` or fake it?
```

Comment:
See https://github.com/ArkEcosystem/core/commit/f8c2acc04a8e9146286ded9c0bb0153d4b79d148 to get an idea how to apply the necessary mocks.

Reply:
That worked with a few tweaks, thanks!